### ISSUE TITLE
The user should be able to zoom in on an image properly

### ISSUE THREAD

**Issue by KirstenDeHaan** [daisy/reading-apps-ux-reqs#11]
Without (or in addition to) using additional accessibility software, a user should be able to zoom in on an image. The image should remain sharp and not lose resolution when scaled or zoomed in.

**Comment by ways2read** [comment]
I think these are helpful additions to go under "The user should have the possibility of enlarging images" in the [images section](https://daisy.github.io/reading-apps-ux-reqs/requirements/draft/#images)

**Comment by mpakarlsson** [comment]
I agree with this requirement, with the exception of the requirement of "remain sharp and not lose resolution". There are technical limitations of the publications image where the image will be unsharp after 100% image enlargement. Suggest to refine this once it goes into the UR document.
